src/app/ngrx/model/podcast.model.ts

```diff
@@ -6,5 +6,4 @@ export interface PodcastModel {
   title: string;
   feederUrl?: string;
   feederId?: string;
-  episodeIds?: number[];
```

Comment:
I didn't actually end up wanting the episodeIds on the podcasts state. It's enough to have the seriesId on the episodes state.